var/spack/repos/builtin/packages/scine-qcmaquis/package.py

```diff
@@ -15,42 +17,63 @@ class ScineQcmaquis(CMakePackage):
     maintainers("adam-grofe")
 
     version("master", branch="master")
+    version("3.1.4", branch="release-3.1.4")
     version("3.1.3", branch="release-3.1.3")
-    version("3.1.2", branch="release-3.1.2", preferred=True)
-    variant(
-        "blas",
-        values=("openblas", "mkl"),
-        default="openblas",
-        description="Which blas library to use.",
-    )
+    version("3.1.2", branch="release-3.1.2")
```

Comment:
Is it deliberate to no longer have a preferred version?

Reply:
I do not see why the old `3.1.2` version should be the preferred one.
I'll let @adam-grofe comment if there was a reason for this choice.

Reply:
If there isn't a reason to use the latest, then I don't see why the latest wouldn't be preferred.

Reply:
Not specifying a preferred one means that the newest version will be chosen, as far as I understand.

Reply:
Yes, I confirmed in the docs. [here](https://spack.readthedocs.io/en/latest/packaging_guide.html#version-selection)